accepted/PSR-7-http-message-meta.md

```diff
@@ -691,3 +691,14 @@ Further characters or sequences in header values should be rejected according
 to the HTTP specification.
 
 [1]: https://datatracker.ietf.org/doc/html/rfc7230#section-3.2
+
+### 7.2 urlencode User Info
```

Comment:
the numbering will conflict with #1296 - i will adjust here once #1296 gets merged